source/ref_trainings.rst

```diff
@@ -32,19 +32,19 @@ General training structure
         #   Name - spec.model.name
         #   Version - spec.model.version
         #   RandomUUID - a random UUID v4, for example be17d12d-df43-4588-99e7-56a0db3cad77
-        artifactNameTemplate: {{ .Name }}-{{ .Version }}-{{ .RandomUUID }}.zip
+        artifactNameTemplate: {{ .Name }}-{{ .Version }}-{{ .RandomUUID }}.zip  # optional
       # The toolchain parameter is a point of extension for different ML frameworks.
       # For now, we only support the Mlfow toolchain
       toolchain: mlflow
       # Mlflow MLProject file can contains the list of entrypoints. You must choose one of these.
       entrypoint: main
-      # Working directory inside a GIT repository
+      # Working directory inside a training (docker) container, which GIT repository copied in.
       workDir: work/dir
-      # The training data for a ML script. You can find find full description :ref:`here <ref_trainings:Training Data>`.
+      # The training data for a ML script. You can find full description there: https://docs.odahu.org/ref_trainings.html#training-data
```

Comment:
Why did you replace dynamic reference with explicit URL?

Reply:
Because dynamic reference doesn't work if it's a part of the code.
![image](https://user-images.githubusercontent.com/49116723/74253107-51ba0500-4cf7-11ea-9bbd-b1e63f3e3723.png)